src/main.rs

```diff
@@ -365,6 +397,57 @@ fn get_help_menu() -> ArgMatches<'static> {
         .arg(Arg::with_name("open")
             .long("open")
             .help("Open the report in a browser once it is complete."))
+        .arg(Arg::with_name("threads")
+            .short("t")
+            .long("threads")
+            .help("Number of threads to use.")
+            .takes_value(true))
+        .arg(Arg::with_name("downloads-folder")
+            .short("d")
+            .long("downloads-folder")
+            .help("Folder where the downloads are stored.")
+            .takes_value(true))
+        .arg(Arg::with_name("dist-folder")
+            .short("D")
+            .long("dist-folder")
+            .help("Distribution folder.")
+            .takes_value(true))
+        .arg(Arg::with_name("results-folder")
+            .short("r")
+            .long("results-folder")
+            .help("Folder where to store the results.")
+            .takes_value(true))
+        .arg(Arg::with_name("apktool-file")
+            .short("a")
+            .long("apktool-file")
+            .help("Path to the apktool file.")
+            .takes_value(true))
+        .arg(Arg::with_name("dex2jar-folder")
+            .short("j")
+            .long("dex2jar-folder")
+            .help("Where to store the jar files.")
+            .takes_value(true))
+        .arg(Arg::with_name("jd-cmd-file")
+            .short("J")
+            .long("jd-cmd-file")
+            .help("Path to the jd-cmd file.")
+            .takes_value(true))
+        .arg(Arg::with_name("results-template")
+            .short("T")
```

Comment:
This will be changed with the new templating system, but as before, no short option and `--template` as the option.